pkg/virtctl/create/vm/vm.go

```diff
@@ -190,27 +199,30 @@ func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmd.Flags().StringArrayVar(&c.containerdiskVolumes, ContainerdiskVolumeFlag, c.containerdiskVolumes, fmt.Sprintf("Specify a containerdisk to be used by the VM. Can be provided multiple times.\nSupported parameters: %s", params.Supported(volumeSource{})))
+	cmd.Flags().StringArrayVar(&c.pvcVolumes, PvcVolumeFlag, c.pvcVolumes, fmt.Sprintf("Specify a PVCs to be used by the VM. Can be provided multiple times.\nSupported parameters: %s", params.Supported(volumeSource{})))
 	cmd.Flags().StringArrayVar(&c.volumeImport, VolumeImportFlag, c.volumeImport, fmt.Sprintf(
-		"Specify the source for DataVolume. Can be provided multiple times.\nSupported parameters:\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s",
+		"Specify the source for DataVolume. Can be provided multiple times.\nSupported parameters:\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s\n  type %s - %s",
 		blank, params.Supported(dataVolumeSourceBlank{}),
 		gcs, params.Supported(dataVolumeSourceGcs{}),
 		http, params.Supported(dataVolumeSourceHttp{}),
 		imageIO, params.Supported(dataVolumeSourceImageIO{}),
-		pvc, params.Supported(dataVolumeSourcePVC{}),
+		pvc, params.Supported(dataVolumeSource{}),
 		registry, params.Supported(dataVolumeSourceRegistry{}),
 		s3, params.Supported(dataVolumeSourceS3{}),
 		vddk, params.Supported(dataVolumeSourceVDDK{}),
-		snapshot, params.Supported(dataVolumeSourceSnapshot{}),
+		snapshot, params.Supported(dataVolumeSource{}),
+		ds, params.Supported(dataVolumeSource{}),
 	))
 
 	cmd.Flags().StringVar(&c.cloudInitUserData, CloudInitUserDataFlag, c.cloudInitUserData, "Specify the base64 encoded cloud-init user data of the VM.")
 	cmd.Flags().StringVar(&c.cloudInitNetworkData, CloudInitNetworkDataFlag, c.cloudInitNetworkData, "Specify the base64 encoded cloud-init network data of the VM.")
 
+	// Deprecated flags
+	cmd.Flags().StringArrayVar(&c.dataSourceVolumes, DataSourceVolumeFlag, c.dataSourceVolumes, "Specify a DataSource to be cloned by the VM. Can be provided multiple times.\nSupported parameters: name:string,src:string,bootorder:uint,size:resource.Quantity\nDEPRECATED: Use --volume-import instead")
+	cmd.Flags().StringArrayVar(&c.clonePvcVolumes, ClonePvcVolumeFlag, c.clonePvcVolumes, "Specify a PVC to be cloned by the VM. Can be provided multiple times.\nSupported parameters: name:string,src:string,bootorder:uint,size:resource.Quantity\nDEPRECATED: Use --volume-import instead")
+	cmd.Flags().StringArrayVar(&c.blankVolumes, BlankVolumeFlag, c.blankVolumes, "Specify a blank volume to be used by the VM. Can be provided multiple times.\nSupported parameters: name:string,size:resource.Quantity\nDEPRECATED: Use --volume-import instead")
```

Comment:
maybe in the hint for the user, you can add the complete command to do the same, e.g. "`volume-datasource` -> Use `volume-import=type:ds` instead."